modal/billing.py

```diff
@@ -0,0 +1,74 @@
+# Copyright Modal Labs 2025
+from datetime import datetime, timezone
+from decimal import Decimal
+from typing import Any, Optional, Union
+
+from modal_proto import api_pb2
+
+from ._utils.async_utils import synchronizer
+from .client import _Client
+from .exception import InvalidError
+
+__all__ = ["workspace_billing_report"]
+
+
+@synchronizer.create_blocking
+async def workspace_billing_report(
+    *,
+    start: Union[datetime, str],  # Start of the report, inclusive
```

Comment:
Personally not a fan of supporting `str` here but I see the value for convenience.

Reply:
I just really hate `datetime.datetime` 😇